os_migrate/plugins/module_utils/reference.py

```diff
@@ -249,6 +249,26 @@ def subnet_pool_ref(conn, id_, required=True):
     return _fetch_ref(conn, conn.network.find_subnet_pool, id_, required)
 
 
+def project_id(conn, ref, required=True):
+    """Fetch ID of a Project identified by reference dict `ref`. Use
+    OpenStack SDK connection `conn` to fetch the info. If `required`,
+    ensure the fetch is successful.
+    Returns: the ID, or None if not found and not `required`
+    Raises: openstack's ResourceNotFound when `required` but not found
+    """
+    return _fetch_id(conn, conn.identity.find_project, ref, required)
+
+
+def project_ref(conn, id_, required=True, get_project_info=False):
```

Comment:
We should remove that parameter from this function signature and always pass False below. See flavor_ref on line 225:

https://github.com/os-migrate/os-migrate/pull/269/files#diff-b38d550e0c19cfb70943298e7af3a33fR225